### ISSUE TITLE
Player who has executed the ban is not shown on the ban screen

### ISSUE THREAD

**Issue by MarioG1** [grandwazir/BanHammer#57]
1.) Ban someone.
2.) Join with the banned account and check out the ban screen.

![Ban Screen](http://s16.postimg.org/qxbotce9h/Unbenannt.png)

There is a similar problem on the temp ban screen

![Ban Screen](http://s13.postimg.org/7oivwc6lj/Unbenannt.png)

**Comment by grandwazir** [comment]
This worked fine when I tested it; I took screenshots for the documentation.

I'll look into it.

**Comment by MarioG1** [comment]
Can you confirm this problem on your own server??

**Comment by grandwazir** [comment]
It works fine for me. Have you by any chance overridden the localisation file? Do you have a localisation.properties in your BanHammer folder?

**Comment by MarioG1** [comment]
I don't have a localisation.propertiesm file in my banhammer folder.

EDIT 1: I've created the file manually bu it is still not working. I also tried an fresh install on my test server with the same result no  localisation.properties is created and the banscreen displays the same things as above.

I haven't mentioned this before but the ban screen shows the correct info's on the first disconnect (If you ban a player who is online he gets the correct info on his disconnect scree but if he tries to join again he gets the buggy ban screen)

EDIT 2: I found the reason of this problem :). If the server runs in offline mode the ban screen is broken  if it switch to online mode everything is working fine.
